### PR TITLE
Adding goldilocks chart

### PR DESCRIPTION
**Pull request by sudermanjr**

### CODE REVIEW

**Review by dosullivan — APPROVED** [review]
This all looks good to me, templated correctly on my machine. Would it be appropriate to include a note in the goldilocks readme that it has dependency on the VPA?

**Comment by sudermanjr** [comment]
e2e tests on this one are going to be difficult due to the depency on VPA.  I will do some thinking on it.

**Comment by sudermanjr** [comment]
@bambash I'm going to need your review on this one until the CODEOWNERS file is merged.

**Pull request merged**
